FirebaseRemoteConfig/Sources/RCNConfigExperiment.h

```diff
@@ -23,14 +23,15 @@
 @interface RCNConfigExperiment : NSObject
 
 /// Designated initializer;
-- (instancetype)initWithDBManager:(RCNConfigDBManager *)DBManager
-             experimentController:(FIRExperimentController *)controller NS_DESIGNATED_INITIALIZER;
+- (nullable instancetype)initWithDBManager:(RCNConfigDBManager *_Nullable)DBManager
+                      experimentController:(FIRExperimentController *_Nullable)controller
+    NS_DESIGNATED_INITIALIZER;
 
 /// Use `initWithDBManager:` instead.
-- (instancetype)init NS_UNAVAILABLE;
+- (nullable instancetype)init NS_UNAVAILABLE;
```

Comment:
I don't think there should be a `nullable` on the NS_UNAVAILABLE init.

Reply:
Should `initWithDbManager` be `nonnull` too?

Reply:
Yes. Some explanation at https://stackoverflow.com/a/42354763/556617